src/components/widgets/filesystem/FileSystem.vue

```diff
@@ -370,6 +405,38 @@ export default class FileSystem extends Mixins(StateMixin, FilesMixin, ServicesM
     return this.$store.state.server.info.registered_directories || []
   }
 
+  transformTimelapseItems (items: AppFile[] | AppDirectory[]) {
+    const timelapses: {[key: string]: AppFile} = {}
+
+    for (const item of items) {
+      if (item.type === 'file' && item.extension !== 'jpg') {
+        timelapses[item.filename.slice(0, -(item.extension.length + 2))] = item
+      }
+    }
+
+    for (const item of items) {
+      if (item.type === 'file' && item.extension === 'jpg') {
+        const name = item.filename.slice(0, -5)
```

Comment:
Should be `slice(0, -4)`:

![image](https://user-images.githubusercontent.com/2041996/160261695-8fde3316-1e2c-484a-b1d2-aa70456850ce.png)